### ISSUE TITLE
ReflectionProperty::isInitialized(): The script tried to access a property on an incomplete object. Please ensure that the class definition

### ISSUE THREAD

**Issue by cuongngoz** [kint-php/kint#393]
Getting this issue after updating from `v3.3` to `v4.2.0` with **PHP 8.1** (previously it worked fine on v3.3 with PHP 7.4)
```
 ErrorException (E_WARNING)
ReflectionProperty::isInitialized(): The script tried to access a property on an incomplete object. Please ensure that the class definition &quot;DoctrineORMModule\Proxy\__CG__\Database\Entity\Options\MySampleEntity&quot; of the object you are trying to operate on was loaded _before_ unserialize() gets called or provide an autoloader to load the class definition

 …/vendor/kint-php/kint/src/Parser/Parser.php447 
```
The issue happens on these codes
```
                $rprop->setAccessible(true);
                if ($rprop->isInitialized($var)) {
                    continue;
                }
```
as I inspected, the `$rprop` value is 
```
ReflectionProperty Object
(
    [name] => __PHP_Incomplete_Class_Name
    [class] => __PHP_Incomplete_Class
)
```
Please help or any advice!!

**Comment by jnvsor** [comment]
Hi @cuongngoz can you confirm that the [phar from 31e9bfa](https://github.com/jnvsor/kint/blob/31e9bfa901dd8ad793a1de88ca5144819783e4b6/build/kint.phar?raw=true) fixes the error before I push this to master?

**Comment by cuongngoz** [comment]
Hi @jnvsor, thanks for the quick support.
Actually I tried by installing dev-master package from your fixed commit (package installed successfully), but got this, pls correct me?
![image](https://user-images.githubusercontent.com/63572184/191701553-d6230a1b-0014-410a-aa66-252cea951208.png)
![image](https://user-images.githubusercontent.com/63572184/191702874-088ae0ce-b869-4ffd-ac9b-c22bd5b77ce8.png)

**Comment by jnvsor** [comment]
That could be your editor's fault. Kint installed through composer `class_alias(Kint::class, 'Kint')` in init.php line 59, so try running it anyway

**Comment by cuongngoz** [comment]
Hello @jnvsor btw, I resolved not only the above `property not found` but also `Class "Kint" not found` issues actually by indicating the class name in autoload and init.php as well, due to installing specific branch/commit version from Github repo package I guess
Then I tested and confirm it works fine now. Very cool, thank you for the fix again.
Just in case, you will push to master and then publish a new release version?
